test/unit/connect_deprish_to_ferc1.py

```diff
@@ -0,0 +1,26 @@
+"""Unit tests for connection between Depreciation and FERC1."""
```

Comment:
Without knowing very much about hooking up the tests, my guess is that you want to start with adding the equivalent for unit tests to this part of `tox.ini`

```
###########################################################################
# Test the code and validate data
###########################################################################
[testenv:integration]
description = Run all the software integration tests
extras =
    test
commands =
    pytest {posargs} {[testenv]covargs} test/integration

[testenv:ci]
description = Run all continuous integration (CI) checks & generate test coverage.
skip_install = false
extras =
    {[testenv:integration]extras}
    {[testenv:linters]extras}
commands =
    coverage erase
    {[testenv:linters]commands}
    {[testenv:integration]commands}
    {[testenv]covreport}
```

Reply:
lol ty! I am still a pytest & tox baby 🍼